queue_job/static/src/views/fields/job_direct_graph/job_direct_graph.scss

```diff
@@ -7,12 +7,4 @@
         height: 100%;
         width: 100%;
     }
-
-    .alert {
-        height: 100%;
-        display: flex;
-        flex-direction: column;
-        align-items: center;
-        justify-content: center;
-    }
-}
```

Comment:
I see you are still using it. Are you sure deleting this won't affect anything?

Reply:
Hi a, I replaced those CSS with explicit classes.

![image](https://github.com/user-attachments/assets/a725dad3-ec6b-440a-b314-4d90e74829ad)